fract4dgui/utils.py

```diff
@@ -21,26 +21,10 @@
 break_new_things = False
 
 
-def threads_enter():
-    if threads_enabled:
-        Gdk.threads_enter()
-
-
-def threads_leave():
-    if threads_enabled:
-        Gdk.threads_leave()
-
-
-def idle_wrapper(callable, *args):
-    threads_enter()
-    callable(*args)
-    threads_leave()
-
-
 def idle_add(callable, *args):
```

Comment:
`idle_add()` is now just mirroring `GLib.idle_add()` which is already used in PNGGen.py and main_window.py. Suggest it can go as well too - a number in gtkfractal.py, doesn't look like it will clash with #175 at a quick read through.

Either way the comment is now incorrect.